ds-caselaw-ingester/lambda_function.py

```diff
@@ -345,10 +369,10 @@ def _build_version_annotation_payload_from_metadata(metadata: dict):
     return payload
 
 
-def get_best_xml(uri, tar, xml_file_name, consignment_reference):
+def get_best_xml(uri, tar: tarfile.TarFile, xml_file_name: str, consignment_reference: str) -> ET.Element:
     xml_file = extract_xml_file(tar, xml_file_name)
     if xml_file:
-        contents = xml_file.read()
+        contents = xml_file.read().decode("utf-8")  # We assume here that our XML is in UTF-8
```

Comment:
Whilst true in practice, I don't think this should be something we have to assume. I think lxml will raise an exception if it receives `str` XML with a document type preamble, since it's now not accurately describing the document.